static/js/admin/src/AddFileForm.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Implements functionality and UI for the generic Ingest Package form
```

Comment:
Apparently this comment has made it into a few files.  Maybe give it something that relates to this specifc form?